packages/ttpassgen/PKGBUILD

```diff
@@ -0,0 +1,31 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=TTPassGen
```

Comment:
always downcase in pkgname. you can add a `_pkgname` var if you need to use the mixed case name somewhere in the script. So do:

```
pkgname=ttpassgen
_pkgname=TTPassGen
```

Reply:
Will include $_pkgname as url is capitalized. Thank you for this note. Maybe update template ?

Reply:
In most case pkgname = _pkgname so no.